docs/configuration/network_filters/mongo_proxy_filter.rst

```diff
@@ -25,6 +26,32 @@ access_log
   path is specified no access logs will be written. Note that access log is also gated by
   :ref:`runtime <config_network_filters_mongo_proxy_runtime>`.
 
+fault
+  *(optional, object)* If specified, the filter will inject faults based on the values in the object.
+
+Fault configuration
+-------------------  
+
+Configuration for MongoDB fixed duration delays. Delays are applied to the following MongoDB operations: Query, Insert,
+GetMore, KillCursors. First command that arrives with no active delays enabled will get the full delay duration and
```

Comment:
The second sentence isn't very clear. It could be
`Once an active delay is in progress, all incoming data up until the timer event fires will be apart of the delay.`

Reply:
And expand that the stat only increments on the delay event being started not per operation.

Reply:
yes, will explain more on stats.

Reply:
should be addressed in the last iteration.